src/controllers/UserController.ts

```diff
@@ -1,52 +1,75 @@
 import { Request, Response } from 'express';
 
 import UserService from '../services/User.service';
+import Api404Error from '../utils/hendleError/hendleError';
+import { UserType } from '../types/user';
+import { logError } from '../utils/hendleError/helpers';
 
 
 export default class UserController {
     constructor(public readonly userService: UserService) {
     }
 
-    public getAutoSuggestUsers = async (req: Request, res: Response): Promise<void> => {
+    public getAutoSuggestUsers = async (req: Request, res: Response, next: (arg: any) => void): Promise<void> => {
```

Comment:
Next function has a better type included in express `import { NextFunction, Request, Response } from 'express';`